lib/logger.js

```diff
@@ -3,7 +3,7 @@ var winston = require('winston');
 var logger = new (winston.Logger)({
     transports: [
       new (winston.transports.Console)({
-          level: 'info',
+          level: 'error',
```

Comment:
Then we should review current logging if it's too noisy. I think that if something is logged at info level, it is important enough to be seen